File: examples/hello.go
package main

import (
	"fmt"
	"time"
)

func main() {
	fmt.Println("Hello from dzjjy deployment!")
	fmt.Println("Application started at:", time.Now().Format("2006-01-02 15:04:05"))

	// 保持运行
	for {
		fmt.Println("Running...", time.Now().Format("15:04:05"))
		time.Sleep(5 * time.Second)
	}
}

File: examples/hello.py
#!/usr/bin/env python3
import time
from datetime import datetime

def main():
    print("Hello from dzjjy deployment!")
    print(f"Application started at: {datetime.now().strftime('%Y-%m-%d %H:%M:%S')}")

    # 保持运行
    while True:
        print(f"Running... {datetime.now().strftime('%H:%M:%S')}")
        time.sleep(5)

if __name__ == "__main__":
    main()
